plugins/source/aws/docs/tables/aws_elasticache_events.md

```diff
@@ -0,0 +1,20 @@
+# Table: aws_elasticache_events
+
+https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_Event.html
+
+The composite primary key for this table is (**date**, **source_identifier**, **source_type**).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|region|String|
+|date (PK)|Timestamp|
```

Comment:
Does this have enough resolution to be unique? Can two events happen at the same time?

Reply:
I don't know... All of the other fields can be empty... So not great candidates for PK... Maybe Hash the object unless you have another idea

Reply:
Yeah let's rather hash the object if we're not certain 👍